common/api-review/functions.api.md

```diff
@@ -44,6 +44,7 @@ export function httpsCallableFromURL<RequestData = unknown, ResponseData = unkno
 // @public
 export interface HttpsCallableOptions {
     timeout?: number;
+    useLimitedUseAppCheckToken?: boolean;
```

Comment:
API council has settled on the simpler name `limitedUseAppCheckTokens` (note the plural) to eliminate the double "use". Please make this change throughout.